compose.yaml

```diff
@@ -10,7 +9,7 @@ services:
     <<: *net-config
     devices:
       - ${MANIPULATOR_SERIAL:?err}:/dev/manipulator
-    volumes: 
+    volumes:
```

Comment:
Recently, rosbot xl has a launch `combined.launch.py` which starts microros and the rosbot_xl_ros driver. 

I see that Dominik also added [here](https://github.com/husarion/rosbot-xl-autonomy/blob/ros2router/compose.yaml) a special launch `/husarion_utils/rosbot.launch.py` for Dockers, which runs healtcheck_node, thanks to which the processes are launched one by one (there is no sudden CPU load)